etools-loading-behavior.html

```diff
@@ -81,6 +82,10 @@
         }
 
       }
+    },
+    clearLoadingQueue: function(event) {
+      event.stopImmediatePropagation();
+      this.globalLoadingElement.messages = [];
```

Comment:
I'm not completely comfortable with this, i would like to take the time and change all write operations on globalLoadingElements.messages to use .set(). Is that ok?